client/graphics/shader.hpp

```diff
@@ -140,9 +140,9 @@ namespace tec {
 		* \param const std::string filename The filename from were the source comes (only for debug purposes)
 		* \return void
 		*/
-		void LoadFromString(const ShaderType type, const std::string& source, const std::string& filename);
+		void LoadFromString(const Shader::ShaderType type, const std::string& source, const std::string& filename);
 
-		GLuint program;
+		GLuint program{ 0 };
```

Comment:
I don't particularly like this. What's wrong with `GLuint program = 0;`?

Reply:
The reason being it is the now recommended way to do initialization https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax

Reply:
Lol these guidelines were written by insane people.